package.json

```diff
@@ -37,14 +37,14 @@
     "@fontsource/pt-sans": "^4.5.11",
     "@fontsource/pt-serif": "^4.5.11",
     "@manuscripts/assets": "^0.6.4",
-    "@manuscripts/body-editor": "1.5.0",
-    "@manuscripts/json-schema": "^2.2.0",
-    "@manuscripts/library": "1.2.0",
+    "@manuscripts/body-editor": "1.5.0-LEAN-2852-3",
+    "@manuscripts/json-schema": "2.2.0-LEAN-2852-5",
+    "@manuscripts/library": "1.1.12-LEAN-2852-7",
     "@manuscripts/quarterback-types": "^1.0.0",
-    "@manuscripts/style-guide": "1.4.0",
+    "@manuscripts/style-guide": "1.4.0-LEAN-2852-1",
     "@manuscripts/title-editor": "^1.1.0",
```

Comment:
Can we get rid of this now?

Reply:
done for article-editor but we still use it for body-editor